docs/components/SideBar/SideBar.jsx

```diff
@@ -20,6 +20,7 @@ export default function SideBar({className}) {
         <NavLink href="/design/typography">Typography</NavLink>
         <NavLink href="/design/sizing">Sizing</NavLink>
         <NavLink href="/design/page-layout">Page Layout</NavLink>
+        <NavLink href="/design/compound-form">Compound Form</NavLink>
```

Comment:
They're ranked by order of importance in the design section instead of alphabetical @amir-clever.

If we aren't sub-navving the layouts, we might want to make a Layout section similar to how we have LESS section.